### DISCUSSION TITLE
Published in Anaesthesia

### DISCUSSION THREAD

**Discussion by mattmoo** [lindeloev/mcp#128] [Show and tell]
https://doi.org/10.1111/anae.15554

(And free access for a week)

![image](https://user-images.githubusercontent.com/20387030/127941698-4b58a2c7-e520-4b7a-a492-e630451ad48f.png)

**Comment by lindeloev** [comment]
Nice use case, congratulations! When mcp 1.0 is out, I'll update the mcp paper, including inspiring use cases. So this is great. 

Do you have a link to code and/or data?

**Reply by mattmoo** [reply]
Probably not unfortunately, the data are secondary personal health data, so simply not redistributable under the ethical approval. The code is a bit of a dog's breakfast dues to protracted development. Itis available [here](https://github.com/mattmoo/checkwho_analysis), but some of the intensive `mcp` stuff was shifted out of this pipeline.
